net-mgmt/telegraf/src/opnsense/service/templates/OPNsense/Telegraf/telegraf.conf

```diff
@@ -127,6 +127,12 @@
 {%   if helpers.exists('OPNsense.telegraf.output.elastic_url') and OPNsense.telegraf.output.elastic_url != '' %}
   urls = ["{{ OPNsense.telegraf.output.elastic_url }}"]
 {%   endif %}
+{%   if helpers.exists('OPNsense.telegraf.output.elastic_username') and OPNsense.telegraf.output.elastic_username != '' %}
+{%   if helpers.exists('OPNsense.telegraf.output.elastic_password') and OPNsense.telegraf.output.elastic_password != '' %}
```

Comment:
the second if clause also please 2 additional indents :)

Reply:
Correct in 497d0e8?